server/index.js

```diff
@@ -1,12 +1,113 @@
 const express = require('express');
+const axios = require('axios');
+const cors = require('cors');
 const app = express();
 const port = 3000;
 
 app.use(express.json());
+app.use(cors());
 app.use(express.urlencoded({ extended: false }));
 app.use('/items/:itemId', express.static('client'));
 
 app.listen(port, () => {
 	// eslint-disable-next-line no-console
 	console.log(`Listening at http://localhost:${port}`);
 });
+
+app.get('/serviceBundles', (req, res) => {
+	let images = axios.get('http://localhost:3006/items/1/bundle.js');
+	let shopping = axios.get('http://localhost:3004/items/1/bundle.js');
+	let reviews = axios.get('http://localhost:3002/items/1/bundle.js');
+	let seller = axios.get('http://localhost:3005/items/1/bundle.js');
+
+	// ****add images service bundle when ready to promise
+	Promise.all([images, shopping, reviews, seller])
```

Comment:
If the 1st promise rejects, it won't wait for the rest before failing

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

You probably want Promise.allSettled() instead here